templates/15-rust.sh

```diff
@@ -94,15 +92,21 @@ build_rust_build() {
 			|| abdie "Compilation failed: $?."
 	else
 		abinfo 'Using fallback build method ...'
-	  if [[ "${CARGO_AFTER[@]}" =~ '-p' ]]; then
-		workspace_flag=''
-	  fi
-	  cargo build "${DEFAULT_CARGO_CONFIG[@]}" \
+		# determine if buiding a package
```

Comment:
This is a confusing comment - what else are we building with Autobuild4? Did you mean to say "Determine if we are building a whole Cargo package"?

Reply:
should be "determine if we are building packages  in a workspace"